src/hooks/useFormActions.js

```diff
@@ -26,7 +26,7 @@ export default ({ dispatch, formRef }) => {
   }
 
   const setValues = (valuesOrValuesGetter, options) => {
-    const currentValues = formRef.current.values
+    const currentValues = formRef.current.values || formRef.current.stateRef.current.values
```

Comment:
Why?

Reply:
When I put a break point before this line formRef.current.values is null and the expected data exists in the stateRef

Reply:
I think you are on to something here, however we should remove the `formRef.current.values` part and do the same on line 23 and 17